server/models/dashboard.js

```diff
@@ -12,6 +12,10 @@ const boxesSchema = Joi.array().items(
       room: Joi.string(),
       camera: Joi.string(),
       name: Joi.string(),
+      chartName: Joi.string(),
+      chartType: Joi.string(),
+      chartPeriod: Joi.string(),
+      chartLimitClass: Joi.string(),
```

Comment:
Please use snake_case like everywhere in the API

Reply:
I change this